<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="booksTabBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/book.png</normaloff>:/Icons/book.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="userTabBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/user.png</normaloff>:/Icons/user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsTabBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/Icons/gear.png</normaloff>:/Icons/gear.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Kitaplar</string>
     </attribute>
     <widget class="QLineEdit" name="searchTextLE">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="list_books">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>isbn</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>isim</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>yazar</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>tür</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>yayım tarihi</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>sayfa sayısı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>yayıncı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>kitap numarası</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="searchCatCB">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>isbn</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>isim</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yazar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tür</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yayım tarihi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sayfa sayısı</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yayıncı</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kitap numarası</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Arama Kategorisi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBtn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ara</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Arama Dizisi</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Kullanici">
     <attribute name="title">
      <string>Ödünç Al</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>781</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ödünç Alınacaklar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="borrowBtn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ödünç Al</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ödünç Alınanlar</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ödünçtekiler</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>771</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ödünçteki kitapları iade etmek için kütüphane görevlisine kitapları verebilirsiniz.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
